Transcom/StartUp.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace Transcom
+{
+    class StartUp
+    {
+        static void Main(string[] args)
+        {
+            using var host = HostBuilder.CreateHostBuilder(args).Build();
+            var executor = (Executor)host.Services.GetService(typeof(Executor));
+            executor.Execute();
```

Comment:
You need to add exception handling here. When there is an exception thrown, the application should show a user-friendly error message and not show exception stack traces